### PR TITLE
2024 MCnet addition

### PR DESCRIPTION
**Pull request by 20DM**
Adding another MCnet project for 2024.

### CODE REVIEW

File: _gsocproposals/2024/proposal_MCnetYODA.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+release system through automated unit-test coverage reports
+by the GitLab continuous integration system, and web-based
+visualization of what code areas need better monitoring.
+
+
+## Expected results and milestones
+
+ * Familiarise with the YODA library;
+ * Familiarise with the YODA unit test system, including current CI scripts;
+ * Hook up existing Familiarise with `gcov` for producing coverage report;
```
Comment (vvolkl) [inline _gsocproposals/2024/proposal_MCnetYODA.md:34]:
```suggestion
 * Hook up existing unit tests with `gcov` for producing coverage report;
```
?

Comment (20DM) [reply _gsocproposals/2024/proposal_MCnetYODA.md:34]: Ouff, thanks for spotting! Copy paste melt down 😭

File: _gsocproposals/2024/proposal_MCnetYODA.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ * Familiarise and its visualization tools
```
Comment (vvolkl) [inline _gsocproposals/2024/proposal_MCnetYODA.md:35]:
```suggestion
 * Familiarise with its visualization tools
```
Do you mean that existing visualization tools in YODA or that the student should pick some visualization library to do the coverage reports?

Comment (20DM) [reply _gsocproposals/2024/proposal_MCnetYODA.md]: The latter! Sorry, I should have read this more carefully last night before committing. I removed the rogue bullet points, so hopefully it makes sense now.

**Review by vvolkl — APPROVED** [review]
Looks good to me, Thanks!

File: _gsocproposals/2024/proposal_MCnetYODA.md (hunk 2 above)
Comment (vvolkl) [reply _gsocproposals/2024/proposal_MCnetYODA.md]: Great, now it's very clear, I think.

**Comment by vvolkl** [comment]
@20DM I forgot one thing: Is there someone who could act as co-mentor for this project (even if it is just to fill in during your holidays, it's good practice)?

**Comment by 20DM** [comment]
@vvolkl Yeah, there's a couple of people that come to mind - I'm sure between Andy and me we'll be able to work something out if necessary. Did you need me to specify someone now already?

**Comment by vvolkl** [comment]
Thanks, If it's ok,  I'll add Andy as a placeholder right now, you can then change this anytime.

**Comment by 20DM** [comment]
Fine by me - thank you!

**Pull request merged**